woocommerce/woocommerce
# Update woocommerce admin node version to v16

### All Submissions:






### Changes proposed in this Pull Request:



Currenlty, core node version is v16 as specified in [.nvmrc](https://github.com/woocommerce/woocommerce/blob/trunk/.nvmrc) and [woocommerce/package.json](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/package.json#L75).

This PR updates the node version in the woocommerce admin package.json to fix ` WARN  Unsupported engine: wanted: {"node":">=12.20.1 <15"} (current: {"node":"v16.14.2","pnpm":"6.32.3"})` during `pnpm install`.



### How to test the changes in this Pull Request:

1. Run `nvm install`
2. Reinstall pnpm if you install it via `npm install -g pnpm`
3. Run `pnpm install` -> should not show ` WARN  Unsupported engine ...`
4. Should build Woo admin:  `pnpm nx build woocommerce-admin`
5. Tests should pass: `pnpm nx run-many --target=test --all --exclude=@woocommerce/api,@woocommerce/api-core-tests,@woocommerce/e2e-core-tests,@woocommerce/e2e-environment,@woocommerce/e2e-utils,woocommerce,woocommerce-legacy-assets,@woocommerce/dependency-extraction-webpack-plugin,@woocommerce/admin-e2e-tests,@woocommerce/onboarding,@woocommerce/js-tests, @woocommerce/notices,@woocommerce/tracks --skip-nx-cache`

### Other information:




### Changelog entry

N/A

### FOR PR REVIEWER ONLY:
